src/users/templatetags/user_tags.py

```diff
@@ -51,6 +51,10 @@ def source_display(source_name):
             "name": "HowLongToBeat",
             "logo": "https://howlongtobeat.com/img/icons/favicon-96x96.png",
         },
+        "imdb": {
+            "name": "IMDB",
+            "logo": "/static/img/logo-imdb.svg",
```

Comment:
Have you tested this? I think it should use the Django template tag like this
```suggestion
            "logo": static("img//logo-imdb.svg"),
```

Reply:
Good catch, thanks. I haven't used Django much. I've fixed this